driver-core/src/main/com/mongodb/client/model/expressions/BooleanExpression.java

```diff
@@ -51,8 +52,8 @@ public interface BooleanExpression extends Expression {
      * The {@code left} branch when {@code this} is true,
      * and the {@code right} branch otherwise.
      *
-     * @param left the left branch.
-     * @param right the right branch.
+     * @param left the left value.
+     * @param right the right value.
      * @return the resulting value.
      * @param <T> The type of the resulting expression.
      */
```

Comment:
What if we rename `left` value to `ifTrue` value and `right` to `ifFalse`/`otherwise` value? I'd say that makes using the method more straightforward.